README.md

```diff
@@ -45,82 +64,81 @@ If you plan to develop *cwe_checker*, it is recommended to build it using the pr
 -   [CWE-476](https://cwe.mitre.org/data/definitions/476.html): NULL Pointer Dereference
 -   [CWE-560](https://cwe.mitre.org/data/definitions/560.html): Use of umask() with chmod-style Argument
 -   [CWE-676](https://cwe.mitre.org/data/definitions/676.html): Use of Potentially Dangerous Function
 -   [CWE-782](https://cwe.mitre.org/data/definitions/782.html): Exposed IOCTL with Insufficient Access Control
 
-Please note that some of the above analyses only are partially implemented at the moment. Furthermore, false positives are to be expected due to shortcuts and the nature of static analysis as well as over-approximation. For more information about the individual checks you can look at the [online documentation](https://fkie-cad.github.io/cwe_checker/doc/html/cwe_checker/index.html).
-
-**NOTE:** The new memory checks for CWEs 415 and 416 are still very experimental and are disabled on a standard run. You can try them out using the `-partial=Memory` command line flag.
-
-**NOTE:** We recently decided to deprecate the support for the old emulation based checks for CWEs 415, 416 and 787. In addition to trying out the new memory checks, users of these checks should also take a look at the [BAP toolkit](https://github.com/BinaryAnalysisPlatform/bap-toolkit), which provides better-maintained (and still emulation based) versions of these checks.
+Please note that some of the above analyses only are partially implemented at the moment.
+Furthermore, false positives are to be expected due to shortcuts and the nature of static analysis as well as over-approximation.
 
 ## Integration into other tools ##
-*cwe_checker* comes with scripts for IDA Pro and Ghidra, which parse the output of *cwe_checker* and annotate the found CWEs in the disassembler for easier manual analysis. See the [online documentation](https://fkie-cad.github.io/cwe_checker/doc/html/cwe_checker/index.html#ToolIntegration) for their usage. The IDA Pro plugin also uses colors to represent the  severeness of the found issues (yellow, orange, or red). The following screenshot shows some results:
-
-<p align="center">
-    <img src="doc/images/example_ida_anotation.png" alt="IDA Pro anotation" width="50%" height="50%"/>
-</p>
 
-## How to extend cwe_checker? ##
-You can find some information about how to write your own check [here](https://fkie-cad.github.io/cwe_checker/doc/html/cwe_checker/index.html#HackingHowto)
-
-If you plan to open a PR, please utilize [precommit](https://pre-commit.com) in your development environment to catch many issues before the code review.
-
-### Contribute ###
-Contributions are always welcome. Just fork it and open a pull request!
+*cwe_checker* comes with a script for Ghidra,
+which parses the output of the *cwe_checker* and annotates the found CWEs in the disassembler for easier manual analysis.
+The script is located at `ghidra_plugin/cwe_checker_ghidra_plugin.py`, usage instructions are contained in the file.
```

Comment:
I think a Demo/Screenshot would be cool after all. Breaks up the Wall of text.
Maybe a short screencast instead of a Screenshot to see the Tool in action (incl. its Ghidra integration!

Reply:
You are right. I will look into it. Where would you place it? Right at the beginning in the "What is cwe_checker"-section?

Reply:
I think I'd put it right before the Installation section. So the interested user can read up on what the tool does up to that point and then get a clear image of what it may look like when using the tool.